typescript/infra/config/environments/mainnet3/warp/configGetters/getoUSDTTokenWarpConfig.ts

```diff
@@ -223,29 +253,33 @@ const productionExtraLockboxes = {
         rateLimitPerSecond: productionRateLimitByChain.ethereum,
       },
     },
+    {
+      lockbox: productionCCIPTokenPoolAddresses.ethereum,
+      limits: productionCCIPTokenPoolLimits,
+    },
   ],
-  base: [
```

Comment:
Will this remove the old extra lockboxes? Because there is still some collateral in it

Reply:
good callout